sig-windows/annual-report-2021.md

```diff
@@ -3,120 +3,79 @@
 ## Current initiatives
 
 1. What work did the SIG do this year that should be highlighted?
-
-   -
-   -
-   -
-
+   - Lead the transition to hostProcess containers.
+   - Defined the `kubectl node logs` command interface.
+   - Made the developer UX for windows transparent with sig-windows-dev-tools.
+   - Defined windows operational readiness standards.
+   
 2. What initiatives are you working on that aren't being tracked in KEPs?
-
-   -
-   -
-   -
+   - Migration of the windows kube-proxy to KPNG (led by amim).
+   - Migration of testgrid reporting jobs from aks-engine to cluster-api/cluster-api-provder-azure.
+   - Dockershim removal / validation for Windows nodes.
 
 3. KEP work in 2021 (1.x, 1.y, 1.z):
-
-<!--
-In future, this will be generated from kubernetes/enhancements kep.yaml files
-1. with SIG as owning-sig or in participating-sigs
-2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
--->
-
    - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+     - (1.22) [1122 - windows-csi-support](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1122-windows-csi-support/README.md)
    - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+     - (1.23) [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/README.md)
+     - (1.23) https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission/kep.yaml
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - (1.22) [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/README.md)
+     - (1.23) [2802 -Identify Windows pods at API admission level authoritatively](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission/README.md)
+   - Pre-alpha (Targeting 1.24)
+     - [2578 - Windows Conformance](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/2578-windows-conformance/kep.yaml)
```

Comment:
is this the right title?  I thought we went with "Operational Readiness"

Reply:
Looks like some references got updated (like the k/ehancements issue https://github.com/kubernetes/enhancements/issues/2578) but others (like the key.yaml - https://github.com/kubernetes/enhancements/blob/18b4a6fd320bbbd4faed9a2cf531b5fc358df263/keps/sig-windows/2578-windows-conformance/kep.yaml#L1) did not.

@knabben can you clean up the Windows-conformance references when you get a chance?

Reply:
fixed it to say op-readiness